docs/Configuring_openshift_virtualization_vm_for_dpdk.md

```diff
@@ -96,12 +96,21 @@ spec:
           threads: 2
           cores: 5
           dedicatedCpuPlacement: true
+          isolateEmulatorThread: true
 ...
 ```
 > **Notes**
 > - The sockets field should be 1, in order to make sure the CPUs are scheduled from the same NUMA Node.
 > - In this snippet example, the VM will be scheduled with 5 [hyper-threads](https://access.redhat.com/articles/7445) (or 10 CPUs).
 
+#### SMT Aligment
+On Nodes where SMT is enabled and CPU-Manager's policy is set to `full-pcpus-only`, there is a significance to the amount of CPUs requested. If the requested total CPUs is even, the isolateEmulatorThread feature requires also doing the following in order to avoid `SMT Alignment` errors by:
+- Adding an annotation to the VirtualMachine's spec.template.spec.metadata.annotation: `kubevirt.io/CPUManagerPolicyBetaOptions: full-pcpus-only`
+- Enabling kubevirt's `CPUManagerPolicyBetaOptions` feature-gate:
+  - `oc annotate --overwrite -n openshift-cnv hco kubevirt-hyperconverged kubevirt.kubevirt.io/jsonpatch='[{"op": "add", "path": "/spec/configuration/developerConfiguration/featureGates/-", "value": "CPUManagerPolicyBetaOptions"}]'`
+
+More information in [BZ2228103](https://bugzilla.redhat.com/show_bug.cgi?id=2228103)
```

Comment:
This part is dependent on the backport of https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2616 and might not be needed.
Could you please introduce it in follow-up?